template.nunjucks

```diff
@@ -47,16 +47,62 @@
   {{ ('node_modules/clay-kiln/dist/clay-kiln.js')|includeFile }}
   </script>
 
-  {# templates #}
+  {# template wrappers #}
   <template class="kiln-pane-template">
     <div class="kiln-toolbar-pane-background">
       <div class="kiln-toolbar-pane">
         <header>
           <span class="pane-header"></span>
-          <button class="close">{% include 'public/media/components/clay-kiln/close.svg' %}</button>
+          <button class="close">{% include 'public/media/components/clay-kiln/close.svg' ignore missing %}</button>
         </header>
         <div class="pane-inner"></div>
       </div>
     </div>
   </template>
+
+  <div class="kiln-pane-template-elements">
+
+  {# sub-templates: messages #}
```

Comment:
Confused me slightly that this comment is not indented, but do see your intention.